assets/shaders/CM.frag

```diff
@@ -0,0 +1,55 @@
+#version 300 es
```

Comment:
all .frag files: I'd prefer them in `src/render/shaders/glsl`, then compiled to `src/render/shaders`